credentials/google/insecure/insecure.go

```diff
@@ -0,0 +1,29 @@
+/*
+ *
+ * Copyright 2022 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package insecure defines insecure credentials for google cloud services.
+package insecure
+
+import (
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/xds/bootstrap"
+)
+
+func init() {
+	bootstrap.RegisterCredentials(&insecure.CredsBuilder{})
```

Comment:
I meant this should be in `credentials/insecure` with the creds themselves.

Reply:
Done. This leads to a cyclic dependency in bootstrap_test.go, so changed insecure.Bundle() to nil.